sources/packages/backend/libs/services/src/notifications/notification/notification.service.ts

```diff
@@ -25,13 +25,22 @@ import {
 } from "./gc-notify.model";
 import { CustomNamedError, processInParallel } from "@sims/utilities";
 import { GC_NOTIFY_PERMANENT_FAILURE_ERROR } from "@sims/services/constants";
+import { NotificationMetadata } from "@sims/sims-db/entities/notification-metadata.type";
 
 /**
  * While performing a possible huge amount of inserts,
  * breaks the execution in chunks.
  */
 const NOTIFICATIONS_INSERT_CHUNK_SIZE = 1000;
 
+interface NotificationToSave {
```

Comment:
Just taking a second look at this interface, can you please elaborate on why we need it?